### PR TITLE
Bug 1160505 - Fixing ui tests with long alerts r=kgrandon

### PR DESCRIPTION
**Pull request by albertopq**

### CODE REVIEW

**Comment by lightsofapollo** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=3ce77bd5c69bc2ef9cab81085e1abeb39e2f6b7f

File: apps/system/test/marionette/dialog_style_test.js [hunk 1]
```diff
@@ -27,7 +27,24 @@ marionette('Dialogs', function() {
   });
 
   test('displaying a long alert', function() {
-    dialog.alert('1\n1\n1\n1\n1\n1\n1\n1\n1\n1\n1\n1\n1\n');
+    var msg = '111111111111111111111111111111111111';
```
Comment (KevinGrandon) [inline apps/system/test/marionette/dialog_style_test.js]: Can we use a loop instead? e.g., 

```
for (var i=0;i<100;i++) msg += 'i';
```

**Comment by lightsofapollo** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=f1dd84fb043beaa0059a9d41e42ba869347c6db2

**Comment by lightsofapollo** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=d107ad479793974b7b599272581ba37ce9c43063

**Pull request merged**